src/components/toast/index.tsx

```diff
@@ -0,0 +1,102 @@
+import { useRouter } from 'next/router'
+import { useCallback, useEffect } from 'react'
+import { default as reactHotToast, Toast } from 'react-hot-toast'
+import Toaster from './components/toaster'
+import ToastDisplay from './components/toast-display'
+import { ToastDisplayProps, ToastColor } from './components/toast-display/types'
+
+const AUTO_DISMISS_DEFAULT = 4000
+
+interface toastOptions {
+  /**
+   * Set the number of milliseconds after which to auto-dismiss.
+   * Defaults to 4000. Set to "false" for show-till-dismissed toast.
+   */
+  autoDismiss?: number | boolean
+  /**
+   * Flag that there is interactive content in the toast.
+   * If the toast contains interactive content,
+   * then we treat it with `role="dialog"`.
+   * Note this defaults to `true` if `actions` are provided, `false` otherwise.
+   */
+  isInteractive?: boolean
```

Comment:
🚧 Looks like this can be removed since it's not used in `toast`!

Reply:
✅ 🔪 Removed, thanks for flagging this! 💖